project/profiles/capacity4more/modules/c4m/content/c4m_content/c4m_content.module

```diff
@@ -717,37 +717,63 @@ function c4m_content_create_highlight_link($gid, $nid, $highlight) {
  *   The markup of the tree.
  */
 function c4m_content_taxonomy_build_tree($items, $link_term_to_facet = array()) {
-  // Filter all items that are not leafs at the tree.
-  $filtered_items = $items;
-  foreach($items as $item) {
-    $parent = taxonomy_get_parents($item['tid']);
-    if (!empty($parent)) {
-      // If item has a parent, unset the parent at 'filtered items' array.
-      $parent_id = (string) key($parent);
-      foreach ($filtered_items as $key => $filtered) {
-        if ($filtered['tid'] == $parent_id) {
-          unset($filtered_items[$key]);
-        }
+  // Each item at array represents 2 level tree.
```

Comment:
2 level trees.
// First node is the root (level 1), and all others, if exist, are children.